dev-docs/adunit-reference.md

```diff
@@ -0,0 +1,342 @@
+
+<a name="mediatypes" />
+
+### `adUnit.mediaTypes`
+
+See the table below for the list of properties in the `mediaTypes` object of the ad unit.  For example ad units showing the different media types, see the [Examples](#addAdUnits-Examples) below.
+
+{: .table .table-bordered .table-striped }
+| Name     | Scope                                                        | Type   | Description                                                                                    |
+|----------+--------------------------------------------------------------+--------+------------------------------------------------------------------------------------------------|
+| `banner` | Required, unless either of the other properties are present. | Object | Defines properties of a banner ad.  For examples, see [`adUnit.mediaTypes.banner`](#banner).   |
+| `video`  | Required, unless either of the other properties are present. | Object | Defines properties of a video ad.  For examples, see [`adUnit.mediaTypes.video`](#video).      |
+| `native` | Required, unless either of the other properties are present. | Object | Defines properties of a native ad.  For properties, see [`adUnit.mediaTypes.native`](#native). |
+
+<a name="banner" />
+
+#### `adUnit.mediaTypes.banner`
+
+{: .table .table-bordered .table-striped }
+| Name    | Scope    | Type                                  | Description                                                                             |
+|---------+----------+---------------------------------------+-----------------------------------------------------------------------------------------|
+| `sizes` | Required | Array[Number] or Array[Array[Number]] | All sizes this ad unit can accept.  Examples: `[400, 600]`, `[[300, 250], [300, 600]]`. |
+| `name`  | Optional | String                                | Name for this banner ad unit.  Can be used for testing and debugging.                   |
+
+<a name="native" />
+
+#### `adUnit.mediaTypes.native`
+
+The `native` object contains the following properties that correspond to the assets of the native ad.
+
+{: .table .table-bordered .table-striped }
+| Name          | Scope    | Type                                  | Description                                                                                                                       |
+|---------------+----------+---------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------|
+| `sizes`       | Required | Array[Number] or Array[Array[Number]] | All sizes this ad unit can accept.  Examples: `[400, 600]`, `[[300, 250], [300, 600]]`.                                           |
+| `type`        | Optional | String                                | Used as a shorthand, e.g., `type: 'image'` implies required fields `image`, `title`, `sponsoredBy`, `clickUrl`.                   |
```

Comment:
Can we link to http://prebid.org/dev-docs/show-native-ads.html#pre-defined-native-types here?

Reply:
Great idea, added link in 3525e50